torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -893,7 +1070,18 @@ def crop_bounding_box(
 
 
 def crop_mask(mask: torch.Tensor, top: int, left: int, height: int, width: int) -> torch.Tensor:
-    return crop_image_tensor(mask, top, left, height, width)
+    if mask.ndim < 3:
+        mask = mask.unsqueeze(0)
+        needs_squeeze = True
+    else:
+        needs_squeeze = False
```

Comment:
@pmeier Another bugfix. Without this the new pad can't handle images with no channels:
```
test/test_prototype_transforms_functional.py:236 (TestKernels.test_against_reference[crop_mask-144])
Traceback (most recent call last):
  File "./vision/test/test_prototype_transforms_functional.py", line 241, in test_against_reference
    actual = info.kernel(input, *other_args, **kwargs)
  File "./vision/torchvision/prototype/transforms/functional/_geometry.py", line 1073, in crop_mask
    return crop_image_tensor(mask, top, left, height, width)
  File "./vision/torchvision/prototype/transforms/functional/_geometry.py", line 1045, in crop_image_tensor
    return pad_image_tensor(image[..., max(top, 0) : bottom, max(left, 0) : right], padding_ltrb)
  File "./vision/torchvision/prototype/transforms/functional/_geometry.py", line 876, in pad_image_tensor
    return _pad_with_scalar_fill(image, torch_padding, fill=fill, padding_mode=padding_mode)
  File "./vision/torchvision/prototype/transforms/functional/_geometry.py", line 890, in _pad_with_scalar_fill
    num_channels, height, width = shape[-3:]
ValueError: not enough values to unpack (expected 3, got 2)
```

Reply:
Resolved at #6949